internal/otkdisk/partition.go

```diff
@@ -0,0 +1,70 @@
+package otkdisk
+
+import (
+	"fmt"
+
+	"golang.org/x/exp/slices"
```

Comment:
The future is indeed now, so we can just use `slices` here. Might be worth doing this in a follow-up and changing it globally. :)